_posts/2017-10-20-from-docker-to-moby.markdown

```diff
@@ -0,0 +1,127 @@
+---
+layout: post
+title:  "From Docker to Moby"
+date:   2017-10-20 10:08:27
+categories: [post]
+tags: [docker]
+img: /img/docker.png
+summary: ""
+changefreq: yearly
+---
+At DockerCon 2017 Austin
+[Moby](https://blog.docker.com/2017/04/introducing-the-moby-project/) was the
+big announcement.
+
+It created confusion and some communities are still trying to understand what is
+going on. I think it's time to step back and see what we have after seven months
+for the announcement.
+
+1. `containerd` is living a new life, the first stable release will happen soon.
+   It went donated to CNCF.
+2. `notary` is the project behind `docker trust`. I wrote a full ebook about
+   [Docker Security](https://scaledocker.com) if you need to know more. It's
+   under the CNCF now.
+3. github.com/docker/docker doesn't exist anymore there is a new repository
+   called github.com/moby/moby . It contains the same code but it opens the door
```

Comment:
Wrong information, It does not contain the same code, the repo is transitioning to be a place where one can take the components to assemble container based systems.
Also worth mentioning: https://github.com/docker/docker-ce that is the CE assembly